eth-connector/src/connector_impl.rs

```diff
@@ -84,7 +84,7 @@ impl AdminControlled for EthConnector {
 }
 
 impl EthConnector {
-    pub(crate) fn deposit(&mut self, proof: &Proof) -> Promise {
+    pub(crate) fn deposit(&mut self, proof: Proof) -> Promise {
```

Comment:
Disagree with that changes. `proof` value is borrowed but should not.

Reply:
Is it? It's borrowed in the previous version.